superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx

```diff
@@ -267,7 +267,7 @@ export default class AdhocFilterControl extends React.Component {
         <OnPasteSelect
           isMulti
           name={`select-${this.props.name}`}
-          placeholder={t('choose a column or metric')}
+          placeholder={t('choose one or more column or metric')}
```

Comment:
I don't have strong opinions on whether this should be singular or plural "column/columns" and "metric/metrics", and it seems the grammar guidelines are loose as well.